soda-core/src/soda_core/cli/handlers/contract.py

```diff
@@ -36,10 +36,13 @@ def handle_verify_contract(
     ]
 
     if is_using_remote_contract(dataset_identifiers) and soda_cloud_client:
-        contract_yaml_sources += [
-            YamlSource.from_str(soda_cloud_client.fetch_contract_for_dataset(dataset_identifier))
-            for dataset_identifier in dataset_identifiers
-        ]
+        for dataset_identifier in dataset_identifiers:
+            contract = soda_cloud_client.fetch_contract_for_dataset(dataset_identifier)
+            if not contract:
+                return (
+                    ExitCode.LOG_ERRORS
+                )  # TODO: verify this is OK as it breaks the "multiple contracts" functionality
```

Comment:
do we need to resolve this dilemma since it applies to local verification as well and therefore is a very fundamental decision?